### PR TITLE
Time

### PR DESCRIPTION
**Pull request by pkasinski**
The class Time has been added and code in some classes has been refactored. Moreover, a test for the class Mouse has been added.

### CODE REVIEW

File: core/src/model/monsters/Cyclops.java [hunk 1]
```diff
@@ -10,18 +10,18 @@ public class Cyclops extends Monster {
     private boolean stuckInYDirection = false;
 
     /**
-     * Uses the default constructor of its superclass with default values
+     * Initializes a Cyclops object.
      */
     public Cyclops(float x, float y, World<IEntity> world) {
-        super(x, y, 32, 32, 1, 10, 5, Faction.MONSTER, world,1,5);
+        this(x, y, Faction.MONSTER, world);
```
Comment (rojtekin) [inline core/src/model/monsters/Cyclops.java:16]: why does a initializer exist?

File: core/src/model/monsters/Mouse.java [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
-    private float timeSinceDirectionChanged = 0;
+    private float timeWhenDirectionChanged = 0;
 
     /**
-     * Uses the default constructor of its superclass with default values
+     * Initializes a Mouse object.
      */
     public Mouse(float x, float y, World<IEntity> world) {
-        super(x, y, 16, 16, 2,1,1, Faction.MONSTER, world,1,1);
+        this(x, y, Faction.MONSTER, world);
```
Comment (rojtekin) [inline core/src/model/monsters/Mouse.java:14]: why does a initalizer exist

**Review by rojtekin — APPROVED** [review]
lgtm

**Review by augnor — APPROVED** [review]
LGTm

**Pull request merged**